Security/SessionDownloadStrategy.php

```diff
@@ -39,14 +47,25 @@ class SessionDownloadStrategy implements DownloadStrategyInterface
     protected $sessionKey = 'sonata/media/session/times';
 
     /**
-     * @param TranslatorInterface $translator
-     * @param ContainerInterface  $container
-     * @param int                 $times
+     * @param TranslatorInterface        $translator
+     * @param ContainerInterface|Session $container
+     * @param int                        $times
      */
-    public function __construct(TranslatorInterface $translator, ContainerInterface $container, $times)
+    public function __construct(TranslatorInterface $translator, $container, $times)
```

Comment:
but if you extend it the signature is different...


but indeed, it is different now, too